ports/stm32/qspi.c

```diff
@@ -370,7 +378,7 @@ static int qspi_read_cmd_qaddr_qdata(void *self_in, uint8_t cmd, uint32_t addr,
             | 0 << QUADSPI_CCR_SIOO_Pos // send instruction every transaction
             | 1 << QUADSPI_CCR_FMODE_Pos // indirect read mode
             | 3 << QUADSPI_CCR_DMODE_Pos // data on 4 lines
-            | 4 << QUADSPI_CCR_DCYC_Pos // 4 dummy cycles
+            | (2 * num_dummy) << QUADSPI_CCR_DCYC_Pos // 2N dummy cycles
```

Comment:
Is the num_dummy parameter doubled everywhere it's used? If so, would it be clearer to double the values stored in the chip params, so `num_dummy` is exactly the number of dummy cycles?

Reply:
My mistake, missed that usage when I searched.